experiments/CSS Examples/BANA CSS Draft.css

```diff
@@ -0,0 +1,331 @@
+    text-indent: 8rem; /* Runovers align to cell 9 */
+}
+
+.level-4 .level-2 {
+    padding-left: 2rem; /* Subitems start at cell 3 */
+    text-indent: 6rem; /* Runovers align to cell 9 */
+}
+
+.level-4 .level-3 {
+    padding-left: 4rem; /* Sub-subitems start at cell 5 */
+    text-indent: 4rem; /* Runovers align to cell 9 */
+}
+
+.level-4 .level-4 {
+    padding-left: 6rem; /* Sub-sub-subitems start at cell 7 */
+    text-indent: 2rem; /* Runovers align to cell 9 */
+}
+
+/* Level 5: Five Levels (1-11, 3-11, 5-11, 7-11, 9-11) */
+.level-5 .level-1 {
+    padding-left: 0; /* Main items start at cell 1 */
+    text-indent: 10rem; /* Runovers align to cell 11 */
+}
+
+.level-5 .level-2 {
+    padding-left: 2rem; /* Subitems start at cell 3 */
+    text-indent: 8rem; /* Runovers align to cell 11 */
+}
+
+.level-5 .level-3 {
+    padding-left: 4rem; /* Sub-subitems start at cell 5 */
+    text-indent: 6rem; /* Runovers align to cell 11 */
+}
+
+.level-5 .level-4 {
+    padding-left: 6rem; /* Sub-sub-subitems start at cell 7 */
+    text-indent: 4rem; /* Runovers align to cell 11 */
+}
+
+.level-5 .level-5 {
+    padding-left: 8rem; /* Sub-sub-sub-subitems start at cell 9 */
+    text-indent: 2rem; /* Runovers align to cell 11 */
+}
+
+/* General Exercise Formatting */
+.exercise {
+    margin-top: 1rem; /* Blank line before exercise */
+    margin-bottom: 1rem; /* Blank line after exercise */
+    padding-left: 0; /* Start at cell 1 for main entry */
+    break-inside: avoid; /* Prevent breaking the exercise across pages */
+}
+
+/* Exercise with No Nesting: 1-3 */
+.exercise.level-1 {
+    padding-left: 0; /* Main item starts at cell 1 */
+    text-indent: 2rem; /* Runovers align to cell 3 */
+}
+
+/* Exercise with One Level of Nesting: 1-5, 3-5 */
+.exercise.level-2 {
+    padding-left: 0; /* Main item starts at cell 1 */
+    text-indent: 4rem; /* Runovers align to cell 5 */
+}
+
+.exercise.level-2 .subitem {
+    padding-left: 2rem; /* Subitems start at cell 3 */
+    text-indent: 2rem; /* Runovers align to cell 5 */
+}
+
+/* Page Break Handling for Nested Subitems */
+.exercise .subitem {
+    break-inside: avoid; /* Keep each subitem intact on the same page */
+}
+
+.exercise.level-2 {
+    break-after: avoid; /* Prevent breaking between main item and subitems if possible */
+}
+
+/* Poetry Formatting */
+/* General Poetry Formatting */
+.poetry {
+    margin-top: 1rem; /* Blank line before poem */
+    padding-left: 0; /* Start at cell 1 for main entry */
+    break-inside: avoid; /* Prevent breaking mid-poem if possible */
+}
+
+/* One-Level Poetry: 1-3 Indentation */
+.poetry.level-1 {
+    padding-left: 0; /* Start first line at cell 1 */
+    text-indent: 2rem; /* Runovers start at cell 3 */
+}
+
+/* Two-Level Poetry: 1-5, 3-5 */
+.poetry.level-2 {
+    padding-left: 0; /* First line of main entry at cell 1 */
+    text-indent: 4rem; /* Runovers stay at cell 5 */
+}
+.poetry.level-2 .sublevel-1 {
```

Comment:
What markup does this rely on? I'm seeing yet another pattern here than for lists and exercises.